examples/hata_basic_queue/bot.py

```diff
@@ -172,19 +181,32 @@ async def skip(ctx):
 @bot.commands
 @checks.guild_only()
 async def pause(ctx):
-    """Pause the current song."""
+    """Pause the current track."""
     await ctx.client.data.lavalink.pause(ctx.guild.id)
     await ctx.reply("Paused player")
 
 
 @bot.commands
 @checks.guild_only()
 async def resume(ctx):
-    """Resume playing the current song."""
+    """Resume playing the current track."""
     await ctx.client.data.lavalink.resume(ctx.guild.id)
     await ctx.reply("Resumed player")
 
 
+@lightbulb.check(lightbulb.guild_only)
+@lightbulb.command(name='now-playing', aliases=['np'])
+async def now_playing(self, ctx):
+    """Get the track that is currently playing."""
+    node = await ctx.client.data.lavalink.get_guild_node(ctx.guild_id)
+    # You could create a queue command buy iterating over `node.queue`.
+    if node.queue:
+        current = node.queue[0].track
+        await ctx.reply(f"Now Playing: {current.info.title}")
+    else:
+        await ctx.reply(f"The queue is empty")
```

Comment:
Fixing possible None errors, and show now_playing, not queue[0].
You could also have an empty queue, but have something playing right now.

```suggestion
    node = await ctx.client.data.lavalink.get_guild_node(event.guild_id)
        
    if not node or not node.now_playing:
        await ctx.reply("Nothing is playing at the moment.")
        return

    # for queue, iterate over `node.queue`, where index 0 is now_playing.
    await ctx.reply(f"Now Playing: {node.now_playing.track.info.title}")
```